composer.json

```diff
@@ -19,7 +19,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": ">=7.4, <8.3",
```

Comment:
Version 7.4 of PHP can be dropped because [number is 7 dead](https://www.php.net/supported-versions).

And please follow the previous version constraints `~8.0.0 || ~8.1.0 || ~8.2.0`.